core/vm/evm.go

```diff
@@ -536,26 +537,43 @@ func (evm *EVM) TobinTransfer(db StateDB, sender, recipient common.Address, gas
 
 	if amount.Cmp(big.NewInt(0)) != 0 {
```

Comment:
I suggest we change this logic to something like:

```go
num, den, err := getTobinTax()

if err == nil || num > 0 {
  tobinTax := new(big.Int).Div(new(big.Int).Mul(numerator, amount), denominator)
	evm.Context.Transfer(db, sender, recipient, new(big.Int).Sub(amount, tobinTax))
	evm.Context.Transfer(db, sender, *reserveAddress, tobinTax)  
} else {
  evm.Context.Transfer(db, sender, recipient, amount)
}
```

Reply:
Broke `getTobinTax()` out into its own function which I think will improve readability, wdyt?